content/people/matthew-carson/contents.lr

```diff
@@ -0,0 +1,13 @@
+_template: people.html
+---
+name: Matthew Carson
+---
+institution: Northwestern University
+---
+photo: matthew-carson.jpg
+---
+role: Digital Repositories Manager
```

Comment:
Haha, I did copy your role because I was thinking Matt (@carsonicator) had a similar role to yours, but at our institution. I'll let @carsonicator suggest a more appropriate one.

Reply:
Kristi and I are both project leads for this on the CD2H grant, so "Northwestern Project Lead" would work. Thanks!

Reply:
Switched to that.